src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print(f'x is %s, y is %s, z is "%s"' % (x, y, z))
```

Comment:
printf has separate operators for separate datatypes. You could also write it like this:
print("x is %d, y is %.2f, z is \"%s\"" % (x, y, z))